### ISSUE TITLE
flycontrols create blank screen for me

### ISSUE THREAD

**Issue by jacksonloper** [jovyan/pythreejs#119]
When I run this I get a blank screen.  Replace flycontrols with orbitcontrols and everything is fine. 

```
ball = Mesh(geometry=SphereGeometry(radius=1), 
                                    material=LambertMaterial(color='red'),
                                    position=[2, 1, 0])

scene = Scene(children=[ball, AmbientLight(color='#777777')])

c = PerspectiveCamera(position=[0, 5, 5], up=[0, 0, 1],
                      children=[DirectionalLight(color='white', 
                                                 position=[3, 5, 1], 
                                                 intensity=0.5)])
renderer = Renderer(camera=c, 
                    scene=scene, 
                    controls=[FlyControls(controlling=c)])
display(renderer)
```

 Any tips?

**Comment by vidartf** [comment]
The OrbitControls will always start out looking at [0, 0, 0] (its orbital target). The flycontrols have no such target, and the camera will retain its initial orientation. You should be able to see something if you set the initial camera/object positions to something else. Alternatively you can use the `look_at`/`lookAt` function (don't remember signature) to point the camera towards the zero point.

Note: FlyControls are currently broken in master, but I have a workable fix locally, that I will submit.

**Comment by vidartf** [comment]
This should now be fixed via #122. Closing as resolved, but feel free to reopen if there are still some issues.
